src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Benefit.java

```diff
@@ -153,4 +153,11 @@ public String getBenefitNameDescriptionWithAcronym(boolean isEnglish) {
         return isHasAcronym() ? of(getShortName()) : empty();
     }
 
+    public static boolean isSscsCaseType(String code, SscsType sscsType) {
```

Comment:
Do you need this method? You kind of get it for free. You can just do a  `SscsType.SSCS2.equals(sscsCaseData.getBenefitType(.getSscsType())`

Reply:
We use the isSscs5Case check in multiple places in case-api. I thought it would be cleaner to have this in one place